docs/geedocs/5.3.1/answer/3499964.html

```diff
@@ -23,8 +23,7 @@ <h2>Uninstalling</h2>
 <li>Stop the Google Earth Enterprise Server, if you have a previous version installed and it is running:
 <p><code>/etc/init.d/geserver stop</code></p>
 </li>
-<li>Navigate to the location at which you initially cloned the GEE git repo.</li>
-<li>Within the GEE git repo, navigate to <code>earthenterprise/earth_enterprise/src/installer</code>
-and run the script <code>uninstall_server.sh</code>.</li></ol></div>
-</ul><div class="footer"><p class="BackToTop"><a href="#top_of_file">Back to top</a> <hr /></p> <p class="copyright">&copy;2015 Google</p></div>
+<li>Navigate to the location at which you initially cloned the GEE git repo or to the installation location (<code>/opt/google</code>).</li>
```

Comment:
This doesn't quite make sense to me:
![image](https://user-images.githubusercontent.com/9662346/55897337-2fa2a400-5b75-11e9-9af2-0b4923282ca0.png)
Steps 4 and 5 are redundant. I think they should be combined.